cs/Serializer/VowpalWabbitDefaultMarshallerExt.tt

```diff
@@ -0,0 +1,194 @@
+﻿// --------------------------------------------------------------------------------------------------------------------
```

Comment:
Yes, I already have .tt compilation in place. Will remove it.

From: John <notifications@github.com<mailto:notifications@github.com>>
Reply-To: JohnLangford/vowpal_wabbit <reply@reply.github.com<mailto:reply@reply.github.com>>
Date: Thursday, November 26, 2015 at 13:49
To: JohnLangford/vowpal_wabbit <vowpal_wabbit@noreply.github.com<mailto:vowpal_wabbit@noreply.github.com>>
Cc: Markus Cozowicz <marcozo@microsoft.com<mailto:marcozo@microsoft.com>>
Subject: Re: [vowpal_wabbit] C# refactoring, memory leak fixes, general goodness,... (#863)


In cs/Serializer/VowpalWabbitDefaultMarshallerExt.tt<https://github.com/JohnLangford/vowpal_wabbit/pull/863#discussion_r46002121>:

> @@ -0,0 +1,194 @@
> +// --------------------------------------------------------------------------------------------------------------------


I'm assuming the corresponding 2970 lines .cs file is automatically generated. Right? Can that be left out?

—
Reply to this email directly or view it on GitHub<https://github.com/JohnLangford/vowpal_wabbit/pull/863/files#r46002121>.